BusBoard.ConsoleApp/Helpers.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using RestSharp;
+using RestSharp.Serialization.Json;
+
+namespace BusBoard.ConsoleApp
+{
+    class Helpers
+    {
+        public static List<BusEntry> GetBuses(string busStopCode, RestClient client)
```

Comment:
It's best practice to use `IEnumerable` instead of `List` as the input and output of functions, just to make them more flexible, unless you specifically need a list. You can always call `toList` on an `IEnumerable` if necessary 🙂